README.md

```diff
@@ -222,6 +222,14 @@ The key-binding `C-c C-w` will also toggle Subword Mode on and off.
 
 Viewing and editing build scripts for [Amaka](http://trashofmasters.github.io/amaka/) will automatically enable PHP Mode.
 
+### Insert current class/namespace ###
+
+```el
+(add-hook 'php-mode-hook
```

Comment:
Please use `with-eval-after-load` and `define-set-key` instead of `add-hook` and `local-set-key` if you write such document. Adding anonymous function to hook is bad, because it cannot be removed by `remove-hook`.